static/bidder-info/silvermob.yaml

```diff
@@ -0,0 +1,8 @@
+maintainer:
+  email: "support@silvermob.com"
+capabilities:
+  app:
```

Comment:
Just confirming that you support only app requests, right?

Reply:
Yes, we support only in-app traffic